tests/tests/test-block.ts

```diff
@@ -52,15 +51,15 @@ describeWithMoonbeam("Moonbeam RPC (Block)", `simple-specs.json`, (context) => {
 	});
 
 	let firstBlockCreated = false;
-	step("should be at block 1 after block production", async function () {
+	it("should be at block 1 after block production", async function () {
```

Comment:
Dumb question for my own learning: what is the difference between `step` and `it`?

Reply:
It's an `it` replacement from the `mocha-steps` library. From the docs:

`Global step() function, as a drop-in replacement for it(). Any failing step will abort the parent describe immediately. This is handy for BDD-like scenarios, or smoke tests that need to run through specific steps.`